.github/workflows/ci.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       with:
         fetch-depth: 0
 
-    - name: Setup .NET Core 6.0.x, 7.0.x
+    - name: Setup .NET SDK
       uses: actions/setup-dotnet@v4
       with:
         dotnet-version: |
```

Comment:
Probably better to explicitly install 8.0.x because sometimes the actions base container isn't up to date. Thoughts?

Reply:
I usually don't specify the current version that will be resolved by `global.json` from a DRY perspective, but there's nothing _stopping_ us from doing that if you have a strong preference for explicitness.

The only caveat I would provide to _trying_ to be explicit by listing the version that'll be resolved by `global.json` is that `actions/setup-dotnet` will resolve the version from `global.json` (i.e. no-op'ing if everything is lined up correctly).

Here's a screencap from what `actions/setup-dotnet` did on a previous run for this PR:

[![image](https://github.com/open-feature/dotnet-sdk/assets/21338699/bcad3aa3-3ed3-4390-87b1-886562479817)](https://github.com/open-feature/dotnet-sdk/actions/runs/7494052087/job/20401116920?pr=172#step:3:1)

> [...] because sometimes the actions base container isn't up to date. Thoughts?

Want to respond to this directly because it's a question that I've had to answer before (so I know the sentiment!), but in my experience, there really isn't a case where you end up with the wrong version because the newest version hasn't yet made it into base container ___provided you're using a `global.json` with one of the roll-forward options enabled___ .